### PR TITLE
Move subscriptions and operatorgroups from loki into cluster-scope

### PR DESCRIPTION
**Pull request by larsks**
It turns out the `loki/` application is also trying to set up the
operatorgroup for the `openshift-redhat-operators` namespace.

This commit moves both subscriptions and operatorgroups out of loki/ and
into cluster-scope/

### CODE REVIEW

**Review by computate — APPROVED** [review]

**Review by computate — APPROVED** [review]
Looks good, I see we are handling the namespaces for openshift-logging and openshift-operators-redhat, as well as their operatorgroups, and the subscriptions of the Logging and Elasticsearch Operator.

**Review by computate — CHANGES_REQUESTED** [review]
Actually, looks like the build is failing trying to the openshift-operators-redhat operatorgroup, but it's actually found here I think: 

[cluster-scope/base/operators.coreos.com/operatorgroups/redhat-operators/operatorgroup.yaml](https://github.com/OCP-on-NERC/nerc-ocp-config/pull/106/files#diff-02f6abf427b9e4f87d0c67b0718366e23262623e9202119509a8f625ef39517b)

**Review by computate — APPROVED** [review]

**Comment by larsks** [comment]
No, I just neglected to include something in my commit :)

File: cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-apiVersion: operators.coreos.com/v1
```
Comment (naved001) [inline cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml:1]: don't we need the apiVersion for operatorgroups?

Comment (dystewart) [inline cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml:1]: Was removal of apiversion here intentional?

Comment (larsks) [reply cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml:1]: we do!

**Review by naved001 — APPROVED** [review]

File: cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml (hunk 1 above)
Comment (larsks) [reply cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml:1]: It was not!

**Pull request merged**